Makefile

```diff
@@ -44,6 +44,10 @@ all: nfs
 sanity-test: nfs
 	./test/sanity/run-test.sh
 
+.PHONY: integration-test
```

Comment:
does `integration-test` work now?

Reply:
if it works, need to add to CI test.

Reply:
OGM, I pushed to the wrong branch. Sorry, I am still testing it.